src/lib/Navigation.svelte

```diff
@@ -10,6 +89,29 @@
 			<div class="mzp-c-navigation-logo">
 				<a href="/">data@mozilla</a>
 			</div>
+			<nav class="mzp-c-menu mzp-is-basic">
+				<ul class="mzp-c-menu-category-list">
+					{#each navigations as navigation}
```

Comment:
There is a gap between the navbar hover item and the content that appears on hover, so if you are too slow with your mouse movement, the content disappears before you reach it. I think there are a few possible solutions (e.g., move the content up, make the hover area larger on the navbar items), any of which would be fine.